### PR TITLE
CI: Update bear-docs with every merge

### PR DESCRIPTION
**Pull request by adtac**

### CODE REVIEW

File: .ci/update_docs.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+cd bear-docs
+git checkout pre
+
+python3 ./generate local
+
+git add -A
+git commit -m "Docs Update"
+
+git remote add github "https://$GITHUB_USERNAME:$GITHUB_PASSWORD@github.com/coala/bear-docs.git"
```
Comment (sils) [inline .ci/update_docs.sh]: rather use a token than usename and password

File: .ci/update_docs.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+cd ~
+rm -rf bear-docs
+
+git clone https://github.com/coala/bear-docs.git
+
+cd bear-docs
+git checkout pre
+
+python3 ./generate local
```
Comment (sils) [inline .ci/update_docs.sh]: you should probably check here if anything actually changed

File: .ci/update_docs.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+source ../rultor_secrets.sh
+
+set -x
+set -e
```
Comment (sils) [inline .ci/update_docs.sh]: the sets are to prevent the source from showing the passwords, right? So they should be *before* the sourcing, right?

**Comment by sils** [comment]
oh right

**Comment by adtac** [comment]
ack https://github.com/coala/coala-bears/pull/983/commits/c0bd42418781157f0888874dd5552f640eccf294

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/983/commits/c1cd3815ccd0e33efe31d7231c1c4110eda12b08

**Comment by sils** [comment]
@rultor merge

**Pull request merged**